docs/src/content/docs/for-administrators/setup-with-k3d-and-nginx.mdx

```diff
@@ -75,6 +75,30 @@ echo "Traefik is running."
 
 ```
 
+<Aside>
```

Comment:
I think in other places we've used this https://starlight.astro.build/guides/authoring-content/#note-aside

```suggestion
:::note
```